cmake/FindZoltan2.cmake

```diff
@@ -33,22 +33,19 @@ endif()
 # Look for the header file
 FIND_PATH(Zoltan2_INCLUDE_DIR NAMES Zoltan2_PartitioningSolution.hpp
           PATH_SUFFIXES trilinos
-          HINTS ${Zoltan2_ROOT}/include)
+          HINTS ${Zoltan2_ROOT}/include $ENV{ZOLTAN2_ROOT}/include)
 
 # Look for the library
 if(NOT BUILD_SHARED_LIBS)
   FIND_LIBRARY(Zoltan2_LIBRARY NAMES libzoltan2.a libtrilinos_zoltan2.a
                PATH_SUFFIXES trilinos
-               HINTS ${Zoltan2_ROOT}/lib)
+               HINTS ${Zoltan2_ROOT}/lib $ENV{ZOLTAN2_ROOT}/lib)
 else()
   FIND_LIBRARY(Zoltan2_LIBRARY NAMES zoltan2 trilinos_zoltan2
                PATH_SUFFIXES trilinos
-               HINTS ${Zoltan2_ROOT}/lib)
+               HINTS ${Zoltan2_ROOT}/lib $ENV{ZOLTAN2_ROOT}/lib)
 endif()
 
-set(Zoltan2_LIBRARIES ${Zoltan2_LIBRARY} )
-set(Zoltan2_INCLUDE_DIRS ${Zoltan2_INCLUDE_DIR} )
-
```

Comment:
I followed this:
https://cmake.org/Wiki/CMake:How_To_Find_Libraries#Writing_find_modules